src/multirust

```diff
@@ -1341,6 +1352,18 @@ show_override() {
     fi
 }
 
+show_toolchain() {
+    local _toolchain=
+    if find_override; then
+        _toolchain="$RETVAL_TOOLCHAIN"
+    elif find_default; then
+        _toolchain="$RETVAL_TOOLCHAIN"
+    else
+        _toolchain="No default"
```

Comment:
For comparison, `rbenv local` prints nothing to STDOUT, but it does print an error to STDERR if it cannot determine the local version of ruby that should apply.